06-air-photos.Rmd

```diff
@@ -0,0 +1,164 @@
+
+##### Q3. For each of the following scales, what is the equivalent ground area in hectares (ha) for a region covering 5”x 5” (Hint: 1" = 2.54 cm; 1 ha = 10,000 m^2^): 1:10,000. 1:12,000, 1:30,000 {.unnumbered}
+
+**Step 2:** The following equation is used to calculate the nominal scale of an air photo. 
+
+$\ S_{p} = f/(H-h_{avg})$
+
+where f is the focal length of the camera, H is the height of the aircraft above sea level and H~avg~ is the average elevation of the terrain in the photo. 
+
+For example: If f = 305 mm, H = 7000 m, h~avg~ = 700 m
+
+$\ S_{p} = 0.305 / (7000 - 700)$
+
+$\ S_{p} = 0.00004841$
+
+Note for the above calculation we first convert focal length in mm to m. 
+
+Next to get the nominal scale in terms of 1:XXXX we need to convert 0.00004 to a fraction. 
+
+First, rewrite 0.00004 as **4/100000**. (i.e. put the 4 in the numerator and then a 1 with the same number of 0's as decimal places in the denominator). 
+
+Then divide 100,000 by 4 to get the nominal scale of 1:25000.  
+
+Answer the following questions for the air photo pair: **BCB93024 124** and **BCB93024 123**
+
+##### Q4. What year were the photos taken? What is the focal length of the camera? What is the scale of the photos?{.unnumbered}
+
+##### Q5. What is the name of the mountain range in the photos? {.unnumbered}
+
+##### Q6. The scale given in the image metadata uses the **average** elevation of the photo. Why would a single number representing photo scale be problematic in this terrain? {.unnumbered}
+
+##### Q7. The highest peak in the photo is at 2030 m. What is the nominal scale for a point located at the summit, assuming the aircraft flew at an altidue of 6393 m? (Hint: Remember to convert the focal length of the camera in mm to m!) {.unnumbered}
+
+------------------------------------------------------------------------
+
+## Task 2: Interpreting air photos {.unnumbered}
+
+**Step 1:** Select an air photo pair with mountainous topography. Get your TA to help you view them in 3D using the stereoscope. 
+
+##### Q8. Using the air photos and Google Earth as a reference, try to recreate the topography in the Virtual Sandbox. Include a photo of one of the air photos and a photo of the sandbox in your final deliverables. {.unnumbered}
+
+**Step 2:** Select an air photo with urban/natural features. Note the roll and photo numbers. 
+
+##### Q9. On your selected photo list at least 4 land cover types (i.e. closed forest, open forest, urban, water, agricultural, herbaceous, shrub) and 2-3 specific urban features (i.e. roads, buildings, docks, seismic lines etc...) present in the photo. {.unnumbered}
+
+##### Q10. For each of the land cover types and features you listed in the previous question, describe how you can identify them in the air photos. How do they vary in terms of texture, shade and shape? {.unnumbered}
+
+------------------------------------------------------------------------
+
+## Task 3: Georeferencing {.unnumbered}
```

Comment:
As discussed, if the physical air photos we have are all in the same location, then we can create a DEM and add an orthorectification step. For the basemap, we might consider using OpenStreetMap.